Mail/Views/Thread List/DraggedEnvelopeView.swift

```diff
@@ -0,0 +1,41 @@
+//
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import MailCore
+import SwiftUI
+
+struct DraggedEnvelopeView: View {
+    @AppStorage(UserDefaults.shared.key(.accentColor)) private var accentColor = DefaultPreferences.accentColor
+    let ammount: Int
+
+    var body: some View {
+        Image(systemName: "envelope.fill")
+            .font(.system(size: UIFontMetrics.default.scaledValue(for: 24)))
+            .padding(value: .small)
+            .overlay(alignment: .bottomTrailing) {
+                if ammount > 1 {
+                    Text("\(ammount)")
+                        .padding(.horizontal, value: .small)
+                        .padding(.vertical, value: .extraSmall)
+                        .background(accentColor.primary.swiftUIColor, in: .circle)
```

Comment:
Because the AccentColor doesn't look really gerat, maybe we can use the same strategy of Apple Mail :
- background : Color.red
- foreground: Color.primary

(don't forget to remove the `@AppStorage`)